examples/knative/powercli/kn-pcli-tag/handler.ps1

```diff
@@ -0,0 +1,64 @@
+Function Process-Handler {
+   param(
+      [Parameter(Position=0,Mandatory=$true)][CloudNative.CloudEvents.CloudEvent]$CloudEvent
+   )
+
+   # Decode CloudEvent
+   $cloudEventData = $cloudEvent | Read-CloudEventJsonData -ErrorAction SilentlyContinue -Depth 10
+   if($cloudEventData -eq $null) {
+      $cloudEventData = $cloudEvent | Read-CloudEventData
+   }
+
+   if(${env:FUNCTION_DEBUG} -eq "true") {
+      Write-Host "DEBUG: K8s Secrets:`n${env:TAG_SECRET}`n"
+
+      Write-Host "DEBUG: CloudEventData`n $(${cloudEventData} | Out-String)`n"
+   }
+
+   if(${env:TAG_SECRET}) {
+      $jsonSecrets = ${env:TAG_SECRET} | ConvertFrom-Json
+   } else {
+      Write-Host "K8s secrets `$env:TAG does not look to be defined"
+      break
```

Comment:
This will return from the handler, correct? Why not use try/catch logic like in the other places of this fn?

Reply:
That's correct. I'll add try/catch instead